test/metabase/query_processor_test/explicit_joins_test.clj

```diff
@@ -387,22 +380,24 @@
       (tt/with-temp Card [{card-id               :id
                            {source-query :query} :dataset_query
                            source-metadata       :result_metadata} (qp.test-util/card-with-source-metadata-for-query
-                                                                    (mt/mbql-query checkins
+                                                                    (mt/mbql-query venues
```

Comment:
I've changed this query because the self join was confusing and was detracting from what's really the point.